modules/system/assets/ui/vendor/focus-visible/focus-visible.js

```diff
@@ -0,0 +1,299 @@
+/**
+ * Release Version v5.0.0
+ * Allow October CMS to work properly with screen readers. 
+ * Applies the :focus-visible polyfill at the given scope.
+ * A scope in this case is either the top-level Document or a Shadow Root.
+ *
+ * @param {(Document|ShadowRoot)} scope
+ * @see https://github.com/WICG/focus-visible
+ */
+function applyFocusVisiblePolyfill(scope) {
+  var hadKeyboardEvent = true;
+  var hadFocusVisibleRecently = false;
+  var hadFocusVisibleRecentlyTimeout = null;
+
+  var inputTypesWhitelist = {
+    text: true,
+    search: true,
+    url: true,
+    tel: true,
+    email: true,
+    password: true,
+    number: true,
+    date: true,
+    month: true,
+    week: true,
+    time: true,
+    datetime: true,
+    'datetime-local': true
+  };
+
+  /**
+   * Helper function for legacy browsers and iframes which sometimes focus
+   * elements like document, body, and non-interactive SVG.
+   * @param {Element} el
+   */
+  function isValidFocusTarget(el) {
+    if (
```

Comment:
@ayumihamsaki you could simplify methods return value to:

```
return !!(el &&
                  el !== document &&
                  el.nodeName !== 'HTML' &&
                  el.nodeName !== 'BODY' &&
                  'classList' in el &&
                  'contains' in el.classList);```

Reply:
Might be not that readable :)

Reply:
double check my commit, thanks.